frontend/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "lint:fix": "next lint --fix",
+    "prettier": "prettier --check \"{pages,public,styles}/*.{js,ts,jsx,tsx,css}\"",
```

Comment:
Ah yes I am glad you asked about this because I forgot to mention it in the PR description! So I had what you suggested before and I also just changed it back to that to make sure but I it seems to work locally but then the GitHub checks fail:

![image](https://user-images.githubusercontent.com/30158156/226419756-9d222f95-5fd8-49cf-97c1-d54e59c4c02e.png)

It looked like it had to do with the `.next` folder, so that's why I have it how it is written. Any ideas on how to change this so it includes everything but the GitHub checks pass? Or is this okay?

Reply:
add .next to .prettierignore